### PR TITLE
Chapter 11

### PR DESCRIPTION
**Pull request by DyannT**
## Related Tickets:
- [#Ticket_ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by DyannT** [comment]
ready

**Review by ghost — COMMENTED** [review]
sao nh` file change dữ vậy @_@ thế này m review sao...

**Comment by DyannT** [comment]
ready

File: app/Http/Controllers/PostController.php [hunk 1]
```diff
@@ -8,17 +8,17 @@ class PostController extends Controller
...
     public function create()
     {
-        //
+        return view('auth/pages/post/create');
     }
 
     public function store(Request $request)
     {
-        //
+        dd($request->all());
```
Comment (ghost) [inline app/Http/Controllers/PostController.php]: dd?

File: resources/css/profile.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    transition: all ease-in-out .3s;
+}
+
+.personal-figcaption:hover {
+    text-align: center;
+    opacity: 1;
+    background-color: rgba(0, 0, 0, .5);
+}
+
+.personal-figcaption:hover i{
```
Comment (ghost) [inline resources/css/profile.css]:
```suggestion
.personal-figcaption:hover i {
```

File: resources/css/profile.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.box-action-info a,button{
+    margin: 10px;
+    color: #fff!important;
+}
+@media (max-width: 767px){
```
Comment (ghost) [inline resources/css/profile.css]:
```suggestion
@media (max-width: 767px) {
```

File: resources/views/layouts/navigation.blade.php [hunk 4]
```diff
@@ -1,24 +1,46 @@
...
+                            :active="request()->routeIs('manager.post.index')">
+                            {{ __('Handle') . ' ' . __('Posts') }}
+                        </x-nav-link>
+
+                        <x-nav-link :href="route('reports.index')" :active="request()->routeIs('reports.index')">
+                            {{ __('Reports') }}
+                        </x-nav-link>
+                    @endif
+
+                    @if(Auth::user()->role->slug === 'admin' || Auth::user()->role->slug === 'moderator')
```
Comment (ghost) [inline resources/views/layouts/navigation.blade.php]:
```suggestion
                    @if (Auth::user()->role->slug === 'admin' || Auth::user()->role->slug === 'moderator')
```

**Comment by DyannT** [comment]
ready

**Pull request merged**